README.md

```diff
@@ -0,0 +1,76 @@
+# AppleTun
+
+## Description
+
+A utility for creating and connecting to a local VPN for ios devices allowing HTTP proxy without device supervision for all connections (not limited to Wi-Fi)
+
+## Installation
+
+```shell
+python3 -m pip install -U AppleTun
+```
+
+## Requirements
+
+[StrongSwan](https://github.com/strongswan/strongswan) installation is required.
+Installation of StrongSwan can be done via `brew install strongswan`, any other package manager, or a manual build (incase special flags are required, exporting of session keys to files for example).
+
+Python requirements are stated in `requirements.txt`.
+
+## Usage
+
+```none
+Usage: appletun [OPTIONS] COMMAND [ARGS]...
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  install-profile  Install AppleTun VPN profile (override if already exists)
+  remove-profile   Remove AppleTun VPN profile
+  start            Start AppleTun VPN
+  stop             Stop AppleTun VPN
+```
+
+## Example
+
+In this example, a VPN profile is installed with together with Proxyman certificate for use with the http proxy.
+After the installation the service is started and a connection to the VPN is established.
+
+```shell
+➜  appletun git:(master) ✗ appletun install-profile -C proxyman-ca.pem
+Profile installed, please accept installation on device
+Please allow installed certificate under Settings > General > About > Certificate Trust Settings
+
+➜  appletun git:(master) ✗ appletun start
+installed profile: AppleTunVPN PSK ************ HTTP 192.168.1.2:9090
+VPN Running
+```
+
+The generated `ipsec.conf` configuration would look as follows:
```

Comment:
This a bit "under-the-hood" for simple users. Let's put this into into own `Inner implementation details` subheading or something like that